src/uuid.rs

```diff
@@ -135,13 +135,15 @@
 #[cfg(test)]
 mod test;
 
-use std::fmt::{self, Display};
+use std::{
+    fmt::{self, Display},
+    str::FromStr,
+};
 
 use crate::{
     error::{Error, Result},
     spec::BinarySubtype,
-    Binary,
-    Bson,
+    Binary, Bson,
```

Comment:
```suggestion
    Binary,
    Bson,
```

Looks like rustfmt didn't like this change.